packages/flutter_tools/lib/src/ios/ios_workflow.dart

```diff
@@ -90,6 +93,17 @@ class IOSWorkflow extends DoctorValidator implements Workflow {
       ));
     }
 
+    // Python dependencies installed
+    if (hasPythonSixModule) {
+      pythonStatus = ValidationType.installed;
+    } else {
+      pythonStatus = ValidationType.missing;
+      messages.add(new ValidationMessage.error(
+        'Python installation missing module "six".\n'
+        'Install via \'sudo easy_install six\'.'
```

Comment:
`pip` doesn't appear to be installed by default, though I wouldn't be surprised if most custom installs list it as a dependency.